TheEthicalHackerCup/Assets/Tutorial/Scripts/TutorialTextTyper.cs

```diff
@@ -48,10 +59,19 @@ void handleSkips()
             {
                 this.tt.Skip();
             }
-            else
-            {
-                showText();
-            }
+            showText();
         }
     }
+
+    public static void startTutorial() {
```

Comment:
How were these tested? There are currently no references to the start and end tutorial functions and trying to call startTutorial() a second time in a scene leads to this situation, where progression is impossible
![image](https://user-images.githubusercontent.com/72239525/216714764-443c13f6-8788-4571-80e8-2c1a25fc4c66.png)